library/docs/docs.css

```diff
@@ -0,0 +1,94 @@
+.docs{
+   display: flex;
+   flex-wrap: wrap;
+   padding-left: 1rem;
+   padding-right: 1rem;
+   margin: 1rem auto;
+}
+
+aside {
+   flex-basis:20rem;
+   background-color:var(--Turquoisecolor);
+   height:50rem;
+ }
+ main {
+   flex-grow: 1;
+ }
+ 
+.heading{
+    text-align:center;
+    font-family: 'Rubik', sans-serif;
+    font-size:2rem;
+}
+ .sidebar-nav{
+    padding:0;
+    margin:0;
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+    list-style:none;
+ }
+ .sidebar-nav h3{
+    font-weight:800;
+    font-size:2rem;
+ }
+ .sidebar-nav a{
+    text-decoration:none;
+ }
+ .sidebar-nav .link-text{
+    font-size:1.8rem;
+    margin-top:1.5rem;
+    cursor: pointer;
+    text-decoration:none;
+    color:var(--light);
+    display:flex;
+    justify-content:center;
+    align-items:center;
+    font-weight:800;
+  }
+
+.component-area{
+   margin:2rem;
+   border-radius:7px;
+   box-shadow:0 4px 8px 0 var(--shadow-color1), 0 6px 20px 0 var(--shadow-color2);
+   padding:1rem;
+}
+.component-area .heading{
+   font-size:2.5rem;
+   text-align:left;
+}
+.component-description{
+  font-size:1.8rem;
+  margin:1rem 0px;
+  text-align:left;
```

Comment:
you can use 0px not an issue I can understand that may be your coding style. But its better to use 0 as value rather that 0px. Also there are places where you have used 0 and here you used 0px so its good to follow one practice throughout. You have use 0 at most of places I would recommend you to go ahead using 0.

Reply:
yeah bhushan i will use 0 . thanks for pointing it out.